src/dash-table/derived/cell/wrappers.tsx

```diff
@@ -1,31 +1,26 @@
 import * as R from 'ramda';
 import React from 'react';
 
-import { memoizeAll, memoizeOne } from 'core/memoizer';
+import { memoizeOne } from 'core/memoizer';
+import memoizerCache from 'core/cache/memoizer';
 import { Data, IVisibleColumn, VisibleColumns, ActiveCell, SelectedCells, Datum, ColumnId, IViewportOffset, Presentation } from 'dash-table/components/Table/props';
 import Cell from 'dash-table/components/Cell';
 import isActiveCell from 'dash-table/derived/cell/isActive';
 import isSelectedCell from 'dash-table/derived/cell/isSelected';
-import memoizerCache from 'core/memoizerCache';
 
-type Key = [number, number];
-type ElementCacheFn = (
-    key: Key,
-    active: boolean,
-    classes: string,
-    columnIndex: number,
-    columnId: ColumnId
-) => JSX.Element;
-
-function getter(
-    elementCache: ElementCacheFn,
-    activeCell: ActiveCell,
-    columns: VisibleColumns,
-    data: Data,
-    offset: IViewportOffset,
-    selectedCells: SelectedCells
-): JSX.Element[][] {
-    return R.addIndex<Datum, JSX.Element[]>(R.map)(
+export default () => new Wrappers().get;
+
+class Wrappers {
+    /**
+     * Returns the wrapper for each cell in the table.
+     */
+    get = memoizeOne((
+        activeCell: ActiveCell,
+        columns: VisibleColumns,
+        data: Data,
+        offset: IViewportOffset,
+        selectedCells: SelectedCells
+    ) => R.addIndex<Datum, JSX.Element[]>(R.map)(
```

Comment:
All the wrappers for the table, impacted by the data, structure/pagination/virtualization and user interactions.